src/cimplog-rdk-logger.c

```diff
@@ -83,3 +83,58 @@ const char *rdk_logger_module_fetch(void)
     return NULL;
 }
 
+/*
+*
+* API to log different component logs in a same log file
+*/
```

Comment:
This is actually to log into an additional module other than the primary module of a component.